syncserver/src/server/mod.rs

```diff
@@ -310,7 +314,7 @@ impl Server {
         });
 
         if let Some(keep_alive) = actix_keep_alive {
-            server = server.keep_alive(keep_alive as usize);
+            server = server.keep_alive(Duration::from_secs(keep_alive as u64));
```

Comment:
I'm pretty sure `keep_alive` is in seconds, but I'd love a double check on this one as well.

Reply:
Yes, actix-http 2.2.2 [used Seconds](https://docs.rs/actix-http/2.2.2/actix_http/enum.KeepAlive.html#variant.Timeout)

Reply:
The `keep_alive` method takes a `Duration` so the question's about if `settings.actix_keep_alive` is in seconds -- it is.

We usually document that in the Setting field's docstring but this one isn't documented -- let's clarify that it's seconds [here](https://github.com/mozilla-services/syncstorage-rs/blob/cc8d584/syncserver-settings/src/lib.rs)

Reply:
Good call, I just updated that.